templates/master/01-master-kubelet/_base/files/kubelet.yaml

```diff
@@ -30,9 +30,7 @@ contents:
       RotateKubeletServerCertificate: true
       DownwardAPIHugePages: true
       PodSecurity: true
```

Comment:
```suggestion
```
Sorry I missed this before, `PodSecurity` also needs to be removed for the unit tests to pass.